source/storing-data/archive.rst

```diff
@@ -7,17 +7,19 @@ QIIME 2 stores data in a directory structure called an :term:`Archive`.
 These archives are zipped to make moving data simple and convenient.
 
 The directory structure has a single root directory named with a :term:`UUID` which
-serves as the :term:`identity` of the archive. Additional files and directories
+serves as the :term:`Identity` of the archive. Additional files and directories
```

Comment:
I think case-sensitive glossary terms are new as of sphinx 3.0 - my personal preference is to revert back to case-insensitive terms (if possible), especially for terms like "identity." Otherwise, maybe we can just disable the case warnings?

Reply:
Update:

https://github.com/sphinx-doc/sphinx/issues/7642#issuecomment-766359877

You should be able to go back to case-insensitive matching (whenever new sphinx packages are up)!

Reply:
@thermokarst, I can revert d1974db and 1777412 before this goes to review. I'll clean up l68 of 1777412, though - unless I'm mistaken, it looks as though the order of the terms in that line is wrong. (If we wanted "Payload", we could have just used term:`Payload`, instead of the more explicit syntax.) We should be able to simplify once non-case-sensitive is accepted again.

Reply:
> it looks as though the order of the terms in that line is wrong

Sounds good, thanks for looking into this.